### ISSUE TITLE
Sons of Krast class counts start at 6/6

### ISSUE THREAD

**Issue by IBwWG** [larsiusprime/tdrpg-bugs#839]
0.8.2, win7x64, HD mode@1704x960windowed

![dq-missing-4-classes-defenders](https://cloud.githubusercontent.com/assets/16719964/15787630/a246ab64-29c3-11e6-8041-a01ba0553bf1.png)

I had unlocked but not attempted to play this level on 0.8.1.  Upgraded then went to play, and none of the other four classes have any defenders in them.  Whoops!

**Comment by IBwWG** [comment]
[export-missing-defenders.zip](https://github.com/larsiusprime/tdrpg-bugs/files/298107/export-missing-defenders.zip)

**Comment by IBwWG** [comment]
Thankfully this seems to be limited to The Sons of Krast.

I noticed that when starting the battle, they initially all have 6/6 defenders.  Then after a second or so the four of them jump to 0/6.

**Comment by IBwWG** [comment]
Oh wait, I get it.  Sorry.

**Comment by ManDude** [comment]
Them still showing 6/6 for a split second is a bug, though.

**Comment by IBwWG** [comment]
Ah, OK>  :)

**Comment by larsiusprime** [comment]
Does this sort of "absent characters flash" bug appear on the other levels with absent characters?

- The way out
- Following Zelemir
- The sidequests with Unazra where Slak/Bakal and Slak/Markos are the only ones there?

**Comment by IBwWG** [comment]
Not sure, but I'll check next time I find those...still on my first run through of NG+ :)

**Comment by IBwWG** [comment]
Return to the Shadows (Slak/Markos) also exhibits this bug.

**Comment by IBwWG** [comment]
BTW I didn't understand from the story before Sons of Krast (which I did read, since it was new) that only they would be fighting.  In RttS the story made more sense, except when I got to the battle, which says before you get there, "The mysterious woman returns, abducting Slak and Markos."  Who is that referring to?

**Comment by ManDude** [comment]
It's referring to sidequest 4, where the mysterious woman lures Bakal and Slak into... I can't remember actually.

**Comment by IBwWG** [comment]
Further to that, in sidequest not sure the number, where this mysterious woman is revealed to be the sorceress, that's all that's revealed.  There doesn't seem to be any story to go with it, and only one journal entry added that alludes to her implicitly.  Did I skip a cutscene somewhere?  (I have been skipping them when I thought they were ones I'd read already the first time through...)

**Comment by ManDude** [comment]
That's #874. :)

**Comment by IBwWG** [comment]
Aha!  Thanks :)

**Comment by Geardan** [comment]
I just encountered a similar problem, but more extensive. Finished NG w/ Heroic Mode, Deluxe Ver. (idk if either matter), started NG+, but didn't play first mission. Went back to original game, ALL classes show 0/1 availability on every map I tried. I reloaded, exited game, etc. but no help. After playing first mission in NG+, Slak was available in NG. Sure enough, my class choices in NG are ALL limited to what is available at the point of NG+ I'm in. I dont' think that's intended behavior, but I guess I don't know for sure.
